rc/windowing/wayland.kak

```diff
@@ -45,15 +45,25 @@ The program passed as argument will be executed in the new terminal' \
 }
 complete-command wayland-terminal shell
 
-define-command wayland-focus -params ..1 -docstring '
-wayland-focus [<kakoune_client>]: focus a given client''s window
-If no client is passed, then the current client is used' \
-%{
-    fail There is no way to focus another window on Wayland
+
+# Detect specific window manager functionality, similarly to how detection.kak
+# peruses different environments.
+declare-option -docstring '
+Ordered list of Wayland window manager modules to try and load.' \
+str-list wayland_wm_modules 'wayland-sway' 'wayland-unsupported'
```

Comment:
I am not really keen on that nested approach to wayland wm, I think a simpler alternative would be to just introduce a `sway.kak` windowing module, put it before `wayland.kak` in the checking order, and have it `require-module wayland` if detection succeeds (then overwrite the focus alias with its own implementation).